process/subprocesses/ghsci.py

```diff
@@ -1035,7 +1029,7 @@ def compare(self, comparison, save=True):
         return comparison
 
     def drop(self, table=''):
-        """Attempt to drop results for this study region.  A specific table to drop may be given as an argument, and if no argument is provided an attempt will be made to drop this study region's database."""
+        """Attempt to drop database results for this study region."""

```

Comment:
Why delete the directions here?  There is a purpose for them --- if someone enters r.help() (or something like that) it tells them about the functions and how to use them using these directions.  Sometimes you don't want to drop all the results, that would be overkill and you just want to drop a table --- this tells people how to do this.  I would not make this change.   This pull request really should be restricted to the scope of earth engine indicator integration.